Api/Controllers/AccountController.cs

```diff
@@ -47,6 +52,28 @@ public async Task<IActionResult> DeleteAccount(string id, CancellationToken canc
 
         return Ok();
     }
+    [HttpPut("update-photo")]
+    public async Task<IActionResult> UpdateAccountPhotoAsync(
+        string userId, IFormFile file, CancellationToken cancellationToken)
+    {
+        var user = await _accountService.GetAccountByIdAsync(userId, cancellationToken);
```

Comment:
The entire logic should be implemented in separate command for example `UpdateAccountPhotoCommand` and executed here via Mediator